recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "tomlkit" %}
-{% set version = "0.9.2" %}
+{% set version = "0.11.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tomlkit-{{ version }}.tar.gz
-  sha256: ebd982d61446af95a1e082b103e250cb9e6d152eae2581d4a07d31a70b34ab0f
+  sha256: 61901f81ff4017951119cd0d1ed9b7af31c821d6845c8c477587bbdcd5e5854e
 
 build:
   number: 0
+  skip: True  # [py<36]
   noarch: python
```

Comment:
We are currently not building `noarch` packages. The package will have to be built as an `architecture specific` packages.